packages/py/py.1.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "Zach Shipko <zachshipko@gmail.com>"
+authors: ["Zach Shipko <zachshipko@gmail.com>"]
+homepage: "https://github.com/zshipko/ocaml-py"
+doc: "https://zshipko.github.io/ocaml-py/doc"
+license: "ISC"
+dev-repo: "git+https://github.com/zshipko/ocaml-py.git"
+bug-reports: "https://github.com/zshipko/ocaml-py/issues"
+tags: ["python"]
+
+depends:
+[
+  "ocaml" {>= "4.05.0"}
+  "dune" {build}
+  "ctypes" {>= "0.13.0"}
+  "ctypes-foreign" {>= "0.4.0"}
+]
+
+depexts: [
```

Comment:
could you create a new package `conf-python-3` instead? See `conf-python-2-7` for reference

Reply:
Alright, I just added it. Do you know why it's failing now? I'm not too familiar with the way the `conf-*` package work.